docs/commands.rst

```diff
@@ -127,13 +127,28 @@ Specifying 0 as the target version will revert all migrations.
 
         $ phinx rollback -e development -t 0
 
+To rollback all migrations to a specific date then use the ``--date``
+parameter or ``-d`` for short.
+
+.. code-block:: bash
+
+        $ phinx rollback -e development -d 20120103
```

Comment:
OK so if I understood correctly, it's fine the way it is right now and then later we can add support for `-d/--date` values like `2016-07-20`, `3 hours ago` etc. That it?